internal/builtinfunctions/functions_test.go

```diff
@@ -1012,3 +1013,23 @@ func TestBuildSchemaNames(t *testing.T) {
 		})
 	}
 }
+
+func TestReadEnvVarFunction(t *testing.T) {
+	// Set the env variables to get below.
+	knownPresentEnvVarKey := "test_known_present_env_var_key"
+	knownNotPresentEnvVarKey := "test_known_not_present_env_var_key"
+	knownEnvVarValue := "known value"
+	defaultEnvVarValue := "default"
+	assert.NoError(t, os.Setenv(knownPresentEnvVarKey, knownEnvVarValue))
+	assert.NoError(t, os.Unsetenv(knownNotPresentEnvVarKey))
+	assert.MapContainsKey(t, "getEnvVar", builtinfunctions.GetFunctions())
+	readEnvVarFunction := builtinfunctions.GetFunctions()["getEnvVar"]
```

Comment:
This is trivial, but changing the name of the local variable to match the new function name would be slightly less jarring here.

Reply:
Fixed.